podium/settings.py

```diff
@@ -43,6 +43,8 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'podium.talks',
+
+    'crispy_forms',
```

Comment:
Please list third party / vendor apps before project apps unless the app is specifically required to be last in the list.  This allows the project apps to override templates, management commands, etc.